mod/starred.php

```diff
@@ -11,32 +11,29 @@ function starred_init(App $a) {
 	$starred = 0;
 	$message_id = null;
 
-	if (! local_user()) {
+	if (!local_user()) {
 		killme();
 	}
 	if ($a->argc > 1) {
 		$message_id = intval($a->argv[1]);
 	}
-	if (! $message_id) {
+	if (!$message_id) {
 		killme();
 	}
 
-	$r = q("SELECT `starred` FROM `item` WHERE `uid` = %d AND `id` = %d LIMIT 1",
-		intval(local_user()),
-		intval($message_id)
-	);
-	if (! DBM::is_result($r)) {
+	$item = Item::selectForUser(local_user(), ['starred'], ['uid' => local_user(), 'id' => $message_id]);
```

Comment:
Do we still need the `'uid' => local_user()` condition in `Item::selectForUser`?

Reply:
Yes. We could query content for ```uid = local_user()``` but also ```uid = 0```.

Reply:
Even in `Item::selectForUser(local_user(), ...)`?